test/miscellaneous_tests/compound_macro.jl

```diff
@@ -60,36 +67,7 @@ let
     @test iscompound(rn.NH3_5)
 end
 
-### Test Various Independent Variables ###
-let
-    @variables x y z
-    @species C(t) H(x) N(x) O(t) P(t,x) S(x,y)
-
-    # Checks that wrong (or absent) independent variable produces errors.
-    @test_throws Exception @eval @compound CO2(t,x) ~ C + 2O
-    @test_throws Exception @eval @compound (NH4(s), [output=true]) ~ N + 4H
-    @test_throws Exception @eval @compound (H2O = 2.0) ~ 2H + O
-    @test_throws Exception @eval @compound PH4(x) ~ P + 4H
-    @test_throws Exception @eval @compound SO2(t,y) ~ S + 2O
-    
-    # Creates compounds.
-    @compound CO2 ~ C + 2O
-    @compound (NH4, [output=true]) ~ N + 4H
-    @compound (H2O(t,x) = 2.0) ~ 2H + O
-    @compound PH4(t,x) ~ P + 4H
-    @compound SO2(t,x,y) ~ S + 2O
-    
-    # Checks they have the correct independent variables.
-    @test issetequal(arguments(ModelingToolkit.unwrap(CO2)), [t])
-    @test issetequal(arguments(ModelingToolkit.unwrap(NH4)), [x])
-    @test issetequal(arguments(ModelingToolkit.unwrap(H2O)), [t, x])
-    @test issetequal(arguments(ModelingToolkit.unwrap(PH4)), [t, x])
-    @test issetequal(arguments(ModelingToolkit.unwrap(SO2)), [t, x, y])
-end
-
```

Comment:
Got moved to better place in the same file. Content not actually changed. Sorry that this makes it harder to look, but since we wanted to refactor the tests,it was easier to do this stuff now as well, rather than having to go back and do it all again.

Reply:
This test now start start around line 105